src/styles/blocks/card.scss

```
@@ -0,0 +1,109 @@
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+
+  box-sizing: border-box;
+
+  border-radius: 5px;
+  border: $card-border-width solid #f3f3f3;
+  padding: $card-padding;
+
+  &__image {
+    margin-top: 16px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    text-decoration: none;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-text-color;
+    margin-top: 40px;
+
+    @include blocks-width;
```

this is not correct. Don't hardcode ur width. This must be a dynamic value
For all the cases!
```suggestion
    width: 100%
```